### ISSUE TITLE
Bun prisma segmentation float on install

### ISSUE THREAD

**Issue by tgatev** [oven-sh/bun#15908]
### Relevant log output

```shell
sadis@o57cpqfe:~/sa-dispatcher.com$ bun install
[0.14ms] ".env"
bun install v1.1.41 (b8f28ed8)
  ⚙️  bufferutil [10/10] ============================================================
Bun v1.1.41 (b8f28ed8) Linux x64
Linux Kernel v5.4.0 | glibc v2.31
CPU: sse42 popcnt avx avx2
Args: "node" "scripts/postinstall.js"
Features: jsc transpiler_cache(3) tsconfig tsconfig_paths
Builtins: "bun:main" "node:fs" "node:path" "node:string_decoder" "node:util/types"
Elapsed: 50ms | User: 36ms | Sys: 16ms
RSS: 1.07GB | Peak: 1.09GB | Commit: 1.07GB | Faults: 0

panic(main thread): Segmentation fault at address 0x0
Crashed while visiting /home/sadis/sa-dispatcher.com/node_modules/@prisma/get-platform/dist/chunk-6HZWON4S.js
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

 https://bun.report/1.1.41/ln1b8f28edChgg8//Bo6ssmE+o4Q0iwioDq/9liDqj8zhD+894gDyng4gDkxs5gD4k4q+D_A2AA


error: postinstall script from "@prisma/engines" terminated by SIGILL (Illegal instruction)
Illegal instruction (core dumped)
```

### Stack Trace (bun.report)

Bun v1.1.41 ([`b8f28ed`](<https://github.com/oven-sh/bun/tree/b8f28ed8afd1c2b60568b2b0158d39a674178027>)) on linux x86_64 [RunAsNodeCommand]

Segmentation fault at address 0x00000000

- [`js_parser.zig:4272`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/js_parser.zig#L4272>): `src.js_parser.Parser._parse__anon_141534`
- [`js_parser.zig:3098`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/js_parser.zig#L3098>): `src.cache.JavaScript.parse`
- [`bundler.zig:1408`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bundler.zig#L1408>): `src.bundler.Bundler.parseMaybeReturnFileOnlyAllowSharedBuffer__anon_121993`
- [`bundler.zig:1255`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bundler.zig#L1255>): `src.bundler.Bundler.parseMaybeReturnFileOnly__anon_117135`
- [`module_loader.zig:1656`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/module_loader.zig#L1656>): `src.bun.js.module_loader.ModuleLoader.transpileSourceCode__anon_117079`
- [`module_loader.zig:2367`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/module_loader.zig#L2367>): `Bun__transpileFile`
- [`ModuleLoader.cpp:609`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/bindings/ModuleLoader.cpp#L609>): `Bun::jsFunctionRequireCommonJS`

Features: tsconfig\_paths, jsc, transpiler\_cache, tsconfig, tsconfig

<sub>Sentry Issue: <strong><a href="https://bun-p9.sentry.io/issues/6164946842/">BUN-A3T</a></strong></sub>

**Comment by 0xc0000f** [comment]
use 1.1.40

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway in #15911
